pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - CSS styling
+  - DOM tree
+---
+
+Changes in the content of a [web page][] occur when at least one of the following occurs:
+
+- **visible changes:** Visible changes in content occur when the rendered pixels change in any part of the document that is currently within the viewport or that can be brought into the viewport via scrolling; or
+- **accessibility tree changes:** Changes in the content of the accessibility tree occur when any state, property or event of a node representing an [accessible object][] of the tree changes, or when any node is inserted in, or removed from the tree; or
+- **audible changes:** Audible changes in content occur when the audio rendered by the [web page][] change.
```

Comment:
"The audio changes" => when my computer plays some music, the audio keeps changing (I'm not listening to _4'33''_ that often). I have a hunch that this is not the kind of changes you want to include.
OTOH, given that these changes in content are only considered as reaction to some action, this is probably OK (if an app requires me to move the phone to keep playing music, it definitely fails that rule…)

Reply:
You're right. The definition doesn't handle time-based media as it is. If the app allows me to move forward or backwards an audio track by moving the device then it should fail. But when the audio is playing the audio is always changing (unless it's playing 4'33'' :-) )
This also happens if it used to control a video. The rendered pixels are changing as part of the video playing (in addition to any audio the video has).

I'll try to rewrite, introducing a reference to compare the output against in order to identify changes.